docs/HACKING.md

```diff
@@ -497,6 +497,12 @@ Now create a fake Windows image:
 dd if=/dev/zero of=windows-dd bs=512 count=10000
 ```
 
+Move the image somewhere that the snap can access:
+
+```shell
+mv windows-dd /var/snap/maas/common/windows-dd
+```
+
```

Comment:
Hmm, this would probably need some QA. Should this go to a separate PR?

Reply:
Sure, I've created an issue: https://github.com/canonical-web-and-design/maas-ui/issues/4207 and will remove these changes from this PR.